source/features/profile-gists-link.tsx

```diff
@@ -42,6 +42,14 @@ async function init(): Promise<void> {
 	);
 
 	const navigationBar = (await elementReady('.UnderlineNav-body'))!;
+
+	// If the button already exists, skip adding it
+	const isGistButtonAlreadyAdded = Boolean(select('.UnderlineNav-body [data-tab-item=\'rgh-gists-item\']'));
+	if (isGistButtonAlreadyAdded) {
+		return;
+	}
+
+	// Add gist counter to profile
 	navigationBar.append(link);
```

Comment:
This would become something like:

```js
attachElement({
  anchor: navigationBar,
  append: () => <a/> // etc, all the logic to create the link should be here
})
navigationBar.replaceWith(navigationBar) // This triggers a refresh logic for GitHub’s overlow menu
```


The same goes for the `.append` calls below